### ISSUE TITLE
Keystore alias not supported

### ISSUE THREAD

**Issue by borigas** [MasonAmerica/mason-cli#1]
If a keystore alias other than `CERT` is used, `APK.parse_cert()` doesn't find the certificate in `META-INF` because it's named after the alias and not `CERT.RSA`. This leads to this error when registering an APK:

```
----------- ERROR -----------
File Name: myapp.apk
Details:
  Mason Platform does not currently support v2 signing scheme for APK's.
  Full Apk (v2) signing is included for Nougat devices and an option in
  Android Studio 2.3+. Please select to either sign the APK with both v1
  (Jar Signature) and v2 (Full APK Signature) or v1 alone. For further
  details reference https://source.android.com/security/apksigning/index.html#
-----------------------------
```

**Comment by trevorhalvorson** [comment]
Thanks for the report @borigas.

Which OS are you using? This sounds like an issue with `openssl`/`keytool` not being available. The mason-cli should support keystore aliases as long as the prereqs are installed/setup correctly: http://docs.bymason.com/prereqs/

**Comment by borigas** [comment]
I'm on Windows 10 compiling a Xamarin app. I'm using the Mason CLI from Ubuntu 16.04.3.

If I have a key alias of `mykeyalias`, my apk contains `META-INF/mykeyali.rsa`. https://github.com/MasonAmerica/mason-cli/blob/738e0f18ef051e70b458ea846f7c736c341f2382/masonlib/external/apk_parse/apk.py#L224 assumes that file is always named `CERT.RSA`. I was able to work around the problem by generating a new key with `CERT` as my key alias, but that shouldn't be necessary

**Comment by trevorhalvorson** [comment]
You're correct, but the fallback to `keytool` should handle the case where a `CERT.RSA` file isn't available.

https://github.com/MasonAmerica/mason-cli/blob/738e0f18ef051e70b458ea846f7c736c341f2382/masonlib/external/apk_parse/apk.py#L232

But this isn't ideal and we do have a fix planned for our next release.

**Comment by scorpiodawg** [comment]
Fixed in version 0.95
